### DISCUSSION TITLE
Can add emotions like sadness, joy, and laughter like bark?

### DISCUSSION THREAD

**Discussion by anni112555** [SWivid/F5-TTS#286] [Ideas]
Can add emotions like sadness, joy, and laughter like bark?

**Comment by abhisirka2001** [comment]
https://huggingface.co/spaces/mrfakename/E2-F5-TTS

might be relevant

**Reply by anni112555** [reply]
I know, what I mean is to add instructions such as <angry><laughing> to generate TTS

**Comment by Reza2kn** [comment]
Watch this! 
https://www.youtube.com/watch?v=-brbxJ43F1c
